target_pose_R:
  position: [0.6624722, -0.2999992, 0.2150011]
  orientation:
    - [-1, 3.403356e-10, 9.265359e-05]
    - [9.265359e-05, 3.673205e-06, 1]
    - [6.776264e-21, 1, -3.673205e-06]
target_pose_L:
  position: [0.5300325, 0.3500131, 0.2299987]
  orientation:
    - [-8.898039e-05, -1, 3.672865e-06]
    - [-1, 8.898037e-05, -3.673532e-06]
    - [3.673205e-06, -3.673192e-06, -1]